docs/introduction.org

```diff
@@ -16,7 +16,22 @@ In the sections below, certain aspects of this programming language will be expl
 
 Note that this documentation is generated from emacs with org-mode and that you can find the Github repository [[https://pismice.github.io/HEIG_ZIG/][here]].
 
-** What this documentation is not
+** What this documentation IS
+This documentation is being worked on by 2 students from the HEIG-VD, in the context of a course for one and a Bachelor Thesis for the other.
+The goal of this documentation is to explore the Zig language in-depth, by first learning a few basics of the language we judge interesting/useful, and then by exploring more advanced topics.
+
+This work is mainly aimed at engineers who already have experiences in C but might consider using Zig as a replacement/alternative. If you are a complete beginner, you might still learn a lot by reading this, but we highly encourage you to look at other ressources aimed at beginners aswell.
+
+The big + this project offers compare to other guides/documentations is that here we are going to learn the language with [[https://en.wikipedia.org/wiki/Literate_programming][literate programming]] (learning via interactive code exemples) because *one snippet of code is worth 1000 words*.
```

Comment:
Add citation in citeproc

Reply:
https://github.com/Pismice/HEIG_ZIG/pull/11#discussion_r1607147705